coalton.asd

```diff
@@ -38,7 +38,9 @@
   :depends-on (#:coalton-compiler
                #:coalton/hashtable-shim
                #:trivial-garbage
-               #:alexandria)
+               #:alexandria
+               #+(or sb-thread threads openmcl-native-threads)
```

Comment:
these should use the ASDF feature gates (ASDF has special syntax for this and your conditionals below)